app/styles/modules/_settings.scss

```diff
@@ -573,3 +573,49 @@ section.modal-panel {
     margin-bottom: 30px;
   }
 }
+
+.fxa-radio {
+
+  input {
+    -moz-appearance: none;
+    -ms-appearance: none;
+    -webkit-appearance: none;
+    border: 1px solid #E3E3E3;
```

Comment:
This needs to use SCSS variables, do we already have a colour that we can use from https://github.com/mozilla/fxa-content-server/blob/master/app/styles/_variables.scss or do we need to define a new variable?